src/main/java/httpserver/http/route/RouteHandler.java

```diff
@@ -0,0 +1,43 @@
+package httpserver.http.route;
+
+import httpserver.http.*;
```

Comment:
I tend to prefer individual imports - as it's clearer what the dependencies are (you should be able to update IntelliJ to stop it doing this)